test/quantization/test_quantized_op.py

```diff
@@ -306,35 +306,24 @@ def test_qrelu_leaky(self, X, alpha):
            alpha=st.floats(0.01, 10.0, allow_nan=False, allow_infinity=False))
     def test_qelu(self, X, alpha):
         X, (scale, zero_point, torch_type) = X
+        output_scale = 0.5
+        output_zero_point = 1
```

Comment:
can we use `_test_activation_function` here?

Reply:
took a quick look, this would require `_test_activation_function` making that func a bit more flexible, so if possible I'd recommend that to be a separate PR, to keep this one single purposed.